src/controller/AdminController.ts

```diff
@@ -0,0 +1,25 @@
+import {Request} from "express";
+import Joi from "joi";
+import {asyncHandler} from "../utils/AsyncHandler";
+import {AdminHandler} from "../handler/AdminHandler";
+
+const adminHandler = new AdminHandler();
+
+export const loginSchema = Joi.object().keys({
+  username: Joi.string().required(),
```

Comment:
In the best case we add a regex here to make sure that all passwords and usernames are in the correct format

Reply:
I thought about it and some input on what regex you'd use would be helpful

Reply:
For usernames I would only allow letters